packages/react/src/components/Storage/FileUploader/Previewer/index.tsx

```diff
@@ -89,9 +109,18 @@ export function Previewer({
           {!isLoading && !isSuccess && (
             <>
               <View>
-                <Button size="small" variation="primary" onClick={onFileClick}>
+                <Button
+                  disabled={
+                    fileStatuses.some((status) => status?.error) ||
+                    isEditingNames.some((edit) => edit) ||
+                    remainingFiles().length === 0
```

Comment:
Why is `remainingFiles` a function?

Reply:
This filters out only files that are remaining that have not completed(success). I exported it to a function for reuse.

Reply:
Maybe rename to getRemainingFiles?